nhais-adaptor/adaptor/outgoing/fhir_helpers/tests/test_operation_definition.py

```diff
@@ -0,0 +1,111 @@
+                                                     contained=[],
+                                                     parameter=[])
+
+            self.assertIsInstance(op_def, OperationDefinition)
+            self.assertEqual(op_def.name, "name.of.operation")
+            self.assertEqual(op_def.code, "code.of.operation")
+            self.assertEqual(op_def.date.as_json(), "2019-04-23 09:00:04.159338")
+            self.assertEqual(op_def.contained, None)
+            self.assertEqual(op_def.parameter, None)
+
+        with self.subTest("with one parameter"):
+            some_param = odh.create_parameter_with_binding(name="SOME_NAME", value="SOME_VALUE")
+
+            op_def = odh.create_operation_definition(name="name.of.operation", code="code.of.operation",
+                                                     date_time="2019-04-23 09:00:04.159338",
+                                                     contained=[],
+                                                     parameter=[some_param])
+
+            self.assertEqual(len(op_def.parameter), 1)
+            self.assertIsInstance(op_def.parameter[0], OperationDefinitionParameter)
+
+    def test_create_practitioner(self):
+        practitioner = odh.create_practitioner_resource(resource_id="practitioner-1", national_identifier="1234567",
+                                                        local_identifier="111")
+
+        self.assertIsInstance(practitioner, Practitioner)
+        self.assertEqual(practitioner.id, "practitioner-1")
+        self.assertEqual(practitioner.identifier[0].value, "1234567")
+        self.assertEqual(practitioner.identifier[1].value, "111")
+
+    def test_create_patient(self):
+        patient = odh.create_patient_resource(resource_id="patient-1", nhs_number="NHSNO11111",
+                                              title="Mr", first_name="Peter", last_name="Parker",
+                                              gender="male", date_of_birth="2019-04-23", place_of_birth="Spidey Town",
+                                              address_line_1="1 Spidey Way", city="Spidey Town", postcode="SP1 1AA")
+
+        self.assertIsInstance(patient, Patient)
+
+    def test_get_parameter_value(self):
+        op_param_transaction_number = odh.create_parameter_with_binding(name="some_param_name", value="17")
+
+        op = odh.create_operation_definition(name="some.name",
+                                             code="some.code",
+                                             date_time="2019-04-23 09:00:04.159338",
+                                             contained=[],
+                                             parameter=[op_param_transaction_number])
+
+        self.assertEqual(odh.get_parameter_value(op, "some_param_name"), "17")
+
+    def test_find_resource(self):
+        with self.subTest("When practitioner details are found in the operation definition"):

```

Comment:
I find it bit more descriptive than just a test function name. Also I was thinking it could have additional test for edge cases like not finding a resource or finding multiples that I haven't catered for at the moment.